src/components/StatusBadge/__tests__/index.spec.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import "jest-styled-components";
+import StatusBadge from "../index";
+import Tooltip from "../../Tooltip";
+
+const badges = [
+  { label: "error", color: "#d3d3d3" },
+  { label: "caution", color: "#f0f0f0" }
+];
+
+describe("StatusBadge", () => {
+  it("should match snapshot", () => {
+    const tree = renderer
+      .create(
+        <StatusBadge type="dark" visibleBadges={badges} hiddenBadges={badges} />
+      )
+      .toJSON();
+
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("mouseLeave should set state to isOpened: false", () => {
+    const setStateMock = jest.fn();
+    const instance = renderer
+      .create(
+        <StatusBadge type="dark" visibleBadges={badges} hiddenBadges={badges} />
+      )
+      .getInstance();
+
+    instance.setState = setStateMock;
+    instance.mouseLeave();
```

Comment:
Simulating an event would be better, it gives you the knowledge that your methods are actually connected to markup.